code/go/0chain.net/smartcontract/dbs/event/blobber.go

```diff
@@ -77,6 +77,55 @@ func (edb *EventDb) GetBlobber(id string) (*Blobber, error) {
 	return &blobber, nil
 }
 
+func (edb *EventDb) GetActiveAndStakableBlobbers(pagination common2.Pagination, healthCheckTimeLimit time.Duration) ([]Blobber, error) {
+	now := common.Now()
+	var blobbers []Blobber
+	result := edb.Store.Get().
+		Select("blobbers.*").
+		Table("blobbers").
+		Joins("left join delegate_pools ON delegate_pools.provider_type = 3 AND delegate_pools.provider_id = blobbers.id AND delegate_pools.status = 0").
```

Comment:
jeck if we use index for this join

Reply:
yes, index is already present with name `idx_dprov_active` , def `CREATE INDEX idx_dprov_active ON public.delegate_pools USING btree (provider_id, provider_type, status)`